pkg/rtc/mediatrack.go

```diff
@@ -177,38 +177,41 @@ func (t *MediaTrack) NotifySubscriberNodeMaxQuality(nodeID livekit.NodeID, quali
 	}
 }
 
-func (t *MediaTrack) SignalCid() string {
-	return t.params.SignalCid
+func (t *MediaTrack) HasSignalCid(cid string) bool {
+	ti := t.MediaTrackReceiver.TrackInfoClone()
```

Comment:
since this is read only do we need to clone here?

Reply:
Don't need it. Will change it.